src/App.jsx

```diff
@@ -1,14 +1,29 @@
-import reactLogo from './assets/react.svg'
 import './App.css'
+import Header from './components/Header/Header'
+import Courses from './components/Courses/Courses'
+import CreateCourse from './components/CreateCourse/CreateCourse'
+import { authorsContext, courseListContext, isTogglePage } from './helpers/context'
+import { useState } from 'react'
+import { mockedAuthorsList, mockedCoursesList } from './constants'
```

Comment:
I would rather not use `mockedXXX` in source code, the naming `mockedXXX` is used in test case data.

Reply:
same as above